init.ps1

```diff
@@ -0,0 +1,18 @@
+# Delegate the work to init.cmd and save off the environment it creates to a temp file.

```

Comment:
I thought you had mentioned moving this file into `Scripts\Windows`? How important is it to even have it?